content/pt-BR/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
I think we should change the folder to "pt" only, following the same folder structure as the older content. 
e.g: 
fr
en